src/main/java/com/google/maps/internal/OkHttpPendingResult.java

```diff
@@ -194,6 +201,22 @@ private T parseResponse(OkHttpPendingResult<T, R> request, Response response) th
         return request.retry();
     }
 
+    byte[] bytes = getBytes(response);
+    R resp;
+    String contentType = response.header("Content-Type");
+
+    // Places Photo API special case
+    if (contentType != null &&
+        contentType.startsWith("image") &&
+        responseClass == PhotoRequest.Response.class &&
+        response.code() == 200) {
+      // Photo API response is just a raw image byte array.
+      PhotoResult result = new PhotoResult();
```

Comment:
Converting a `byte[]` into a `java.awt.Image` makes the production code dependent on `javax.image.ImageIO`, if I'm understanding StackOverflow correctly. Given the common case of caching and then serving the image, a `byte[]` and the server reported `Content-Type` serves both of those use cases. I'll add documentation pointing people at Places Library for retrieving images client side.